force-app/main/default/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -608,7 +612,11 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
     }
 
     shouldRemoveFromElevateBatch(gift) {
-        return gift && gift.isAuthorized() && this.isElevateCustomer;
+        return gift &&
```

Comment:
Can we add parentheses here just so it's very clear what's being checked?

Reply:
+1